yt/visualization/plot_window.py

```diff
@@ -2044,124 +2046,128 @@ class AxisAlignedProjectionPlot(ProjectionPlot, PWViewerMPL):
-         =============================================== ===============================
+        The location of the origin of the plot coordinate system. This
+        is typically represented by a '-' separated string or a tuple of
+        strings. In the first index the y-location is given by 'lower',
+        'upper', or 'center'. The second index is the x-location, given as
+        'left', 'right', or 'center'. Finally, whether the origin is
+        applied in 'domain' space, plot 'window' space or 'native'
+        simulation coordinate system is given. For example, both
+        'upper-right-domain' and ['upper', 'right', 'domain'] place the
+        origin in the upper right hand corner of domain space. If x or y
+        are not given, a value is inferred. For instance, 'left-domain'
+        corresponds to the lower-left hand corner of the simulation domain,
+        'center-domain' corresponds to the center of the simulation domain,
+        or 'center-window' for the center of the plot window. In the event
+        that none of these options place the origin in a desired location,
+        a sequence of tuples and a string specifying the
+        coordinate space can be given. If plain numeric types are input,
+        units of `code_length` are assumed. Further examples:
+
+        =============================================== ===============================
+        format                                          example
+        =============================================== ===============================
+        '{space}'                                       'domain'
+        '{xloc}-{space}'                                'left-window'
+        '{yloc}-{space}'                                'upper-domain'
+        '{yloc}-{xloc}-{space}'                         'lower-right-window'
+        ('{space}',)                                    ('window',)
+        ('{xloc}', '{space}')                           ('right', 'domain')
+        ('{yloc}', '{space}')                           ('lower', 'window')
+        ('{yloc}', '{xloc}', '{space}')                 ('lower', 'right', 'window')
+        ((yloc, '{unit}'), (xloc, '{unit}'), '{space}') ((0, 'm'), (.4, 'm'), 'window')
+        (xloc, yloc, '{space}')                            (0.23, 0.5, 'domain')
+        =============================================== ===============================
 
     right_handed : boolean
         Depreceated, please use flip_horizontal callback.
         Whether the implicit east vector for the image generated is set to make a right
         handed coordinate system with a north vector and the normal vector, the
         direction of the 'window' into the data.
     data_source : YTSelectionContainer Object
-         Object to be used for data selection.  Defaults to a region covering
-         the entire simulation.
+        Object to be used for data selection.  Defaults to a region covering
```

Comment:
is this broken ?
I personally don't fancy obscuring information from git blame with this kind of changes.

Reply:
ok then I suggest adding these commits to .git-blame-ignore-revs

Reply:
These changes are unfortunately interwined with other docstring edits that are part of the PR's changes itself in the same commits, so probably best to leave them alone.

Reply:
unfortunate. I'd offer to rewrite this branch' history but it doesn't seem worth the risk.